src/Components/CreateImageWizardV2/utilities/requestMapper.tsx

```diff
@@ -90,12 +95,21 @@ const uploadTypeByTargetEnv = (imageType: ImageTypes): UploadTypes => {
 const getImageOptions = (
   imageType: ImageTypes,
   state: RootState
-): AwsUploadRequestOptions | GcpUploadRequestOptions => {
+): AwsUploadRequestOptions | AzureUploadRequestOptions | GcpUploadRequestOptions => {
   switch (imageType) {
     case 'aws':
       if (selectAwsShareMethod(state) === 'sources')
         return { share_with_sources: [selectAwsSource(state)?.id || ''] };
       else return { share_with_accounts: [selectAwsAccountId(state)] };
+    case 'azure':
```

Comment:
Looking at the original `onSave` function in `CreateImageWizard/CreateImageWizard.js` it seems that on "sources" the returned values are only `source_id` and `resource_group`. Also source should not be an empty string as it's a required field when creating an image with sources.